tweaktweak.html

```diff
@@ -0,0 +1,371 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>🦅tweak</title>
+</head>
+
+<body>
+    <div id="layout">
+        <div class="side"></div>
+        <div class="left-header"><img src="icons/logo-blue.svg"></div>
+        <div class="list-logo">
+            <div class="list-logo1">
+                <ul>
+                    <li>
+                        <div id="home">
+                            <img style="vertical-align:middle" src="icons/home-blue.svg">
+                            <span>Home</span>
+                        </div>
+                    </li>
+
+                    <li>
+                        <div id="explore">
+                            <img style="vertical-align:middle" src="icons/explore-blue.svg">
+                            <span>Explore</span>
+                        </div>
+                    </li>
+                    <li>
+                        <div id="notifications">
+                            <img style="vertical-align:middle" src="icons/notifications-blue.svg">
+                            <span>Notifications</span>
+                        </div>
+
+                    </li>
+                    <li>
+                        <div id="messages">
+                            <img style="vertical-align:middle" src="icons/messages-blue.svg">
+                            <span>Messages</span>
+                        </div>
+
+                    </li>
+                    <li>
+                        <div id="bookmarks">
+                            <img style="vertical-align:middle" src="icons/bookmarks-blue.svg">
+                            <span>Bookmarks</span>
+                        </div>
+
+                    </li>
+                    <li>
+                        <div id="lists">
+                            <img style="vertical-align:middle" src="icons/lists-blue.svg">
+                            <span>Lists</span>
+                        </div>
+
+                    </li>
+                    <li>
+                        <div id="profile">
+                            <img style="vertical-align:middle" src="icons/profile-blue.svg">
+                            <span>Profile</span>
+                        </div>
+
+                    </li>
+                    <li>
+                        <div id="more11">
+                            <img style="vertical-align:middle" src="icons/more-blue.svg">
+                            <span>More</span>
+                        </div>
+                    </li>
+                </ul>
+            </div>
+            <div class="blank-list">
```

Comment:
I'm not really sure what this list is for? Is it to add spacing to the left hand side of your menu? You might have been able to achieve the same effect using `margin-right` on one of your elements within the left hand menu. Or even padding-left on your list